ballerina/build.gradle

```
@@ -47,7 +47,7 @@ def stripBallerinaExtensionVersion(String extVersion) {
 ballerina {
     packageOrganization = packageOrg
     module = packageName
-    testCoverageParam = "--code-coverage --coverage-format=xml"
+    testCoverageParam = "--code-coverage --coverage-format=xml --excludes=\"modules/**/**\""
```

This is not needed since we do not have any modules now
```suggestion
    testCoverageParam = "--code-coverage --coverage-format=xml"
```